lib/modem_attest_token/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MODEM_ATTEST_TOKEN
+	bool "Modem Attestation Token Library"
+	select NRF_MODEM_LIB
+	select AT_CMD
```

Comment:
I think I'd prefer to depend in instead of selecting these libraries. And then, in line with @lemrey's comments, document somewhere that it's expected that the modules are initialized before the attestation API can be used